src/test/java/org/apache/commons/lang3/time/StopWatchTest.java

```diff
@@ -448,4 +451,29 @@ public void testToStringWithMessage() throws InterruptedException {
         final String splitStr = watch.toString();
         assertEquals(12 + MESSAGE.length() + 1, splitStr.length(), "Formatted split string not the correct length");
     }
+
+    @Test
```

Comment:
Test failable code paths.

Reply:
I think my tests cover all lines. I checked the coverage during writing tests in the IDE with the integrated coverage functionality